sites/main-site/src/content/blog/2025/maintainers-minutes-2025-01-31.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Maintainers Minutes: January 2025'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2025-01-31T12:00:00+01:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-26/maintainers_duck.png'
+headerImageAlt: 'Cartoon yellow rubber duck with nf-core logo badge on its body with the nf-core logo.'
+headerImageDim: [250, 250]
+embedHeaderImage: true
+authors:
+    - 'maxulysse'
+label:
+    - 'maintainers'
+---
+
+import Profile from '@components/GitHubProfilePictureExtended.svelte';
+import { Image } from 'astro:assets';
+
+The 'Maintainers Minutes' aims to give further insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly team meetings.
+
+## Overview
+
+In the January meeting we discussed (amongst others) the following topics:
+
+-   [Spring cleaning](#spring-cleaning)
+-   [References](#references)
+-   [stubs](#stubs)
+-   [CI with latest-everything](#ci-with-latest-everything)
+-   [Helpdesk Americas](#helpdesk-americas)
+-   [Default prefixes in modules](#default-prefixes-in-modules)
+-   [Best practices for spring cleaning](#best-practices-for-spring-cleaning)
+
+## Spring cleaning
+
+We decided that the spring cleaning that we planned to do before the hackathon should be done two weeks before the hackathon.
+So it will happen from March 10th to March 14th.
+We decided to extend the duration of the spring cleaning to a full week to give everyone enough time to participate.
+The idea is to clean up issues, PRs, have a look at the repos and open new issues if needed.
```

Comment:
Maybe a expand with a bit more datail, the community might be interested.

E.g. stuff like: we go through common repos, and close PR that are older than X years (After giving a warning), deleting merged branches, we check slack/github teams to make sure they are up to date, archiving pipelines